src/test/features/response/routes/counter-claim.ts

```diff
@@ -28,11 +30,14 @@ describe('Defendant response: counter claim page', () => {
         idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta', 'defendant')
       })
 
+      checkCountyCourtJudgmentRequestedGuardGuard(app, 'get', pagePath)
+      checkCountyCourtJudgmentRequestedGuardGuard(app, 'post', pagePath)
```

Comment:
That is not right place for it. I do not think testing POST requests in `'on GET'` block is what we tend to do.

Reply:
My mistake. This route has no POST handler at all. Removed testing POST then